src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -746,10 +746,10 @@ private void OnWorkspaceChanged(object sender, EventArgs e)
                 }
             }
 
-            private void OnActiveContextChanged(object sender, DocumentEventArgs e)
+            private void OnActiveContextChanged(object sender, DocumentActiveContextChangedEventArgs e)
             {
                 // REVIEW: it would be nice for changed event to pass in both old and new document.
-                OnSuggestedActionsChanged(e.Document.Project.Solution.Workspace, e.Document.Id, e.Document.Project.Solution.WorkspaceVersion);
+                OnSuggestedActionsChanged(e.Workspace, e.CurrentActiveContextDocumentId, e.Workspace.CurrentSolution.WorkspaceVersion);
```

Comment:
was there any discussion on if it's a problem that you are getting the workspace versoin now, instead of at the time the event was created?